### PR TITLE
Operator upgrade documentation for volume health monitor changes

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Operator upgrade documentation for volume health monitor changes

# GitHub Issues

# Checklist:

- [x ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ x] Have you tested whether the hyperlinks are working properly?
- [ x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by panigs7 — DISMISSED** [review]

File: content/docs/csidriver/installation/operator/_index.md [hunk 1]
```diff
@@ -266,25 +266,64 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
...
+      envs:
+    - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+      value: "true"
 ```
-driver:
-    configVersion: v5
-    replicas: 2
-    dnsPolicy: ClusterFirstWithHostNet
-    common:
+   ii. update the sidecar versions:
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:284]: please correct the indentation

Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:284]: Also mention - Add external-health-monitor sidecar to enable health monitor of CSI volumes from Controller plugin

File: content/docs/csidriver/installation/operator/_index.md [hunk 2]
```diff
@@ -266,25 +266,64 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
 
 **NOTES:** 
 1. If you are trying to upgrade the CSI driver from an older version, make sure to modify the _configVersion_ field if required.
-2. The parameter "dnsPolicy: ClusterFirstWithHostNet" should be added as follows:
+2. Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+   To enable this feature, we will have to modify the below block while upgrading the driver.To get the volume health state `value`
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:270]: To get the volume health state `value - > To get the volume health state add external-health-monitor sidecar in the sidecar section and  `value` under controller...

File: content/docs/csidriver/installation/operator/_index.md [hunk 3]
```diff
@@ -266,25 +266,64 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
...
+    - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+      value: "true"
+      dnsPolicy: ClusterFirstWithHostNet
+      node:
+      envs:
+    - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+      value: "true"
 ```
-driver:
-    configVersion: v5
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:272]: Add config version field and set to v2.1.0

Comment (karthikk92) [reply content/docs/csidriver/installation/operator/_index.md:272]: taken care

File: content/docs/csidriver/installation/operator/_index.md (hunk 2 above)
Comment (karthikk92) [reply content/docs/csidriver/installation/operator/_index.md]: taken care

File: content/docs/csidriver/installation/operator/_index.md (hunk 1 above)
Comment (karthikk92) [reply content/docs/csidriver/installation/operator/_index.md]: tc

Comment (karthikk92) [reply content/docs/csidriver/installation/operator/_index.md]: tc

File: content/docs/csidriver/installation/operator/_index.md [hunk 4]
```diff
@@ -266,25 +266,69 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
...
-```
+   ```yaml
+      driver:
+        configVersion: v2.1.0
+   ```
+2. Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+   To enable this feature, we will have to modify the below block while upgrading the driver.To get the volume health state add 
+   external-health-monitor sidecar in the sidecar section and `value`under controller set to true and the `value` under node set 
+   to true as shown below:
+    i. add controller and node section as bellow:
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:277]: add -> Add
bellow -> below

Comment (karthikk92) [reply content/docs/csidriver/installation/operator/_index.md]: tc]

File: content/docs/csidriver/installation/operator/_index.md [hunk 5]
```diff
@@ -266,25 +266,69 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
...
+          envs:
+            - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+              value: "true"
+        dnsPolicy: ClusterFirstWithHostNet
+        node:
+          envs:
+            - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
+              value: "true"
+    ```
+   ii. update the sidecar versions and Add external-health-monitor sidecar to enable health monitor of CSI volumes from Controller plugin:
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md]: update the sidecar versions and Add external-health-monitor sidecar to enable health monitor of CSI volumes from Controller plugin: -> Update the sidecar versions and add external-health-monitor sidecar if you want to enable health monitor of CSI volumes from Controller plugin:

Comment (karthikk92) [reply content/docs/csidriver/installation/operator/_index.md]: tc

**Review by rensyct — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Review by panigs7 — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**